src/components/Tag/Tag.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '../../theme';
+import { get, lodashGet } from '../../utils/utils';
+import { COMMON, COLOR } from '../../utils/constants';
+
+const TagBase = styled.div`
+    // text
+    ${get('typography.body')};
+    color: ${(props) => props.textColor};
+
+    // border
+    border-radius: 4px;
+
+    // display
+    display: inline-flex;
+
+    // align
+    vertical-align: middle;
+    align-items: center;
+
+    // size
+    padding: 0px 8px;
+    height: 28px;
+
+    // color
+    background: ${(props) => props.backgroundColor};
+
+    ${COMMON};
+    ${COLOR};
+`;
+
+const Tag = ({
+  variant, color, theme: propTheme, ...props
+}) => {
+  const defaultBackground = propTheme.colors.bluePalette.primary;
+  const backgroundColor = lodashGet(propTheme, `colors.${color}Palette.${variant}`, defaultBackground);
+  let textColor = propTheme.colors.white;
+
+  // both light green and light yellow tags have dark palette text
+  if (variant === 'light' && (color === 'green' || color === 'yellow')) {
+    textColor = lodashGet(propTheme, `colors.${color}Palette.text`, propTheme.colors.black);
+  }
+
+  return (
+    <TagBase
+      backgroundColor={backgroundColor}
+      textColor={textColor}
```

Comment:
can you have `theme={propTheme}` to TagBase - this allows `get` to get from the theme passed in, or fallback to the default theme

Reply:
oh true good catch, i'll update